locations/spiders/technopolis_bg.py

```diff
@@ -47,5 +47,8 @@ def parse(self, response):
                     item["opening_hours"].add_days_range(day_range(days[0], days[1]), hours[0], hours[1])
                 else:
                     item["opening_hours"].add_range(days[0], hours[0], hours[1])
-            apply_category(Categories.SHOP_ELECTRONICS, item)
             yield item
+
+    def post_process_feature(self, item, feature):
```

Comment:
Nothing is calling `post_process_feature`? The category must be coming from NSI.

Reply:
Thanks, you are right. I've removed the post process method.